docs/modules/customize/pages/content_security_policy.adoc

```diff
@@ -0,0 +1,53 @@
+= Configuring Content Security Policy Parameters
+
+This is a security feature that helps to detect and mitigate certain types of attacks, including Cross Site Scripting (XSS) and data injection attacks.
+By default, the CSP is enabled, and is configured to be as restrictive as possible, having the following default configuration:
+
+[source,ruby]
+....
+  {
+    "default-src" => %w('self' 'unsafe-inline'),
+    "script-src" => %w('self' 'unsafe-inline' 'unsafe-eval'),
+    "style-src" => %w('self' 'unsafe-inline'),
+    "img-src" => %w('self'),
+    "font-src" => %w('self'),
+    "connect-src" => %w('self'),
+    "frame-src" => %w('self'),
+    "media-src" => %w('self')
+  }
+....
+
+In order to customize the CSP we are providing, have 2 options, either by using a configuration key the initializer `config/initializers/decidim.rb` or by setting values in the Organization's system admin.
+
+Using the initializer is the recommended way to customize the CSP when you have multiple organizations sharing the same Decidim instance, sharing the same tools. Fox example, if you are using a custom map provider, you will need to add the domain to the CSP, so that the map can be displayed. In this case, you will need to add the following to your initializer:
```

Comment:
Let's move this to its own subsection. 

Also, there's a typo ("Fox" instead of "For")

```suggestion

=== Initializer

Using the initializer is the recommended way to customize the CSP when you have multiple organizations sharing the same Decidim instance, sharing the same tools. For example, if you are using a custom map provider, you will need to add the domain to the CSP, so that the map can be displayed. In this case, you will need to add the following to your initializer:
```